source/process/working-at-mattermost.rst

```diff
@@ -0,0 +1,73 @@
+
+While our headquarters is in Palo Alto, California, we use Mattermost software to work remotely around the world. 
+
+Top 10 reasons to work for Mattermost: 
+
+1. **Join an outstanding team:** Work with thoughtful, talented, high performance people who care about results. 
+2. **Growth:** Join a growing open source project, a growing business, and a growing company. When you're growing, every day is your best day ever.
+3. **Work remote:** No commute, plus complete control of your environment and freedom to travel.
+4. **Do good:** Build open source software benefiting thousands of companies, research institutions, schools and non-profits.
+5. **Improve a product you use every day:** Your opinion matters. 
+6. **Be recognized:** Working in open source, your contributions are publicly visible and will echo on throughout your career. 
+7. **Focus on results:** Output matters, not hours worked. Enjoy life and don't burn out. 
+8. **Continual development:** Cross-training and rotation builds up everyone on the team. It's core to our culture. 
+9. **Global perspective:** Develop outstanding software in a multi-lingual, multi-cultural community. 
+10. **Friends around the world:** Build relationships with staff and contributors in dozens of countries. 
+
+----------------------------------------------------------
+Working Remotely 
+----------------------------------------------------------
+
+While we're headquartered in Palo Alto, California, like many open source companies, a large portion of Mattermost, Inc. works remotely from around the world. 
+
+This gives us tremendous advantages: 
+
+1. **Access to top talent.** We can hire from anywhere.
+2. **More comfort.** People have 100% control of their surroundings. 
+3. **No commute.** Removing a 30-60 minute commute saves 250-500 hours a year. 
+4. **Time zone coverage.** It's easier to support customers any time of day, without being up at odd hours.
+5. **Results-focused culture.** Output matters more than hours at a desk. 
+
+Running a distributed, high performance organization 
+
+1) Work from anywhere with good Internet
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Working from anywhere means more time with loved ones, no commute, and controlling everything about your workspace. 
+
+The one requirement is having a strong internet connection for occassional video or audio conferences. Ethernet connections are preferred over Wifi, since Wifi can have interference. 
+
+2) Be explicit 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When you send a message, you take someone's full attention, so get across the relevant information as soon as you can.
+
+Don't say "Are you free to talk?", instead say "Catch up on ulimit issue Customer X is having?"
+
+Likewise, over communicate in writing when you can. The more people can read, the faster they can become effective. 
```

Comment:
"Over communicate" is confusing, is that what you meant? Generally over-communicating is referred to as a negative thing, so maybe phrase it differently (eg be thorough, or over communication is preferable to under communication) or add more context on what you mean here